### PR TITLE
fix settings page title is not updating

### LINKED ISSUE #7304: [UI] Settings: Page title not being updated
#### Current Behavior
When a user navigates to the Settings page, the page title should read "Settings", but doesn't. Instead the previously visited page title is used.

#### Desired Behavior
When a user navigates to the Settings page, the page title should display "Settings".

#### Screenshots/Logs
Screenshot 2023-03-07 at 4 40 06 PM

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI
](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Meyazhagan**
**Notes for Reviewers**

This PR fixes #7304 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/MesherySettings.js [hunk 1]
```diff
@@ -206,6 +206,10 @@ class MesherySettings extends React.Component {
     this.systemResetRef = React.createRef();
   }
 
+  componentDidMount() {
+    this.props.updatepagetitle({ title : "Settings" })
+  }
+
```
Comment (8rxn) [inline ui/components/MesherySettings.js:212]: Hey @Meyazhagan ,
Great work on the PR.
I had a some points to consider:
Is there any specific reason as to why this change was made inside the `MesherySettings.js` and not on `settings.js` inside the pages directory.
That seems like a better approach as there's already a line to updatepagepath there and would make it similar to:

https://github.com/meshery/meshery/blob/005fdd9e53fc4653f04b5a480b3f28c226069c11/ui/pages/system/connections.js#LL13C1-L15C9

**Review by 8rxn — CHANGES_REQUESTED** [review]
If there's no specific reason, the code may be shifted to `settings.js` for better maintainability

**Comment by 8rxn** [comment]
@Meyazhagan One more thing While you're at it, The same issue exits on the `/user/preferences` page as well. 

It'd be great if you could add fixes to that too in the same PR.

**Comment by Meyazhagan** [comment]
@8rxn I updated the changes. Take a look

**Review by 8rxn — APPROVED** [review]
looks good to me 🚀

**Review by leecalcote — APPROVED** [review]

**Pull request merged**